grails-app/conf/application.yml

```diff
@@ -256,6 +256,7 @@ grails:
               alwaysUseDefault: true
               defaultTargetUrl: '/'
             interceptUrlMap: [
+                            {pattern: '/role/**',                                       access: ['denyAll']},
                             {pattern: "/static/console/**",                             access: ['ROLE_SUPER_ADMIN']},
```

Comment:
Could you do me the favor and use double quotes for the access role on your change aswel and on the super_admin (code line below your change). Just so we have a unified pattern here.
Is it necessary to set denyAll?